src/pytest_subtests/plugin.py

```diff
@@ -98,6 +100,24 @@ def _from_test_report(cls, test_report: TestReport) -> SubTestReport:
         return super()._from_json(test_report._to_json())
 
 
+def _addSkip(self: TestCaseFunction, testcase: TestCase, reason: str) -> None:
+    if isinstance(testcase, _SubTest):
+        self._originaladdSkip(testcase, reason)  # type: ignore[attr-defined]
```

Comment:
In both case (subTest or not), we all need to call the original addSkip otherwise the reporting is still wrong.